airflow/providers/google/cloud/triggers/dataproc.py

```diff
@@ -178,6 +179,18 @@ def serialize(self) -> tuple[str, dict[str, Any]]:
             },
         )
 
+    def safe_to_cancel(self) -> bool:
```

Comment:
When it is reattempted, does this attached to the existing external job or will it create new job?